intro_modern.tex

```diff
@@ -3,13 +3,19 @@ \chapter{Introduction}
 
 This document is a reference manual for
 \ifthenelse{\boolean{PrintImplementationRq}}%
-{the ACSL implementation provided by the Frama-C
-  framework~\cite{frama-c}.}%
-{ACSL.}
+           {the ACSL implementation\footnote{Into that reference manual of ACSL,
```

Comment:
Minor suggestions: 

`Into that` -> `In the`

And a few lines below:

`implemented into` -> `implemented in`